country.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'country_module'
+class Country
+
+  include Check_country
+
+  def initialize(country_name ,population, gross_domestic_production, states, army_strength, state_of_country)
+    @country_name = country_name
+    @population = population
+    @gross_domestic_production = gross_domestic_production
+    @states = states
+    @army_strength = army_strength
+    @state_of_country = state_of_country
+  end
+
+  def get_name
```

Comment:
Instead of defining getters manually, you could use attr_accessor to make the code more concise.https://medium.com/@rossabaker/what-is-the-purpose-of-attr-accessor-in-ruby-3bf3f423f573